website/partials/navbar.pug

```diff
@@ -1,18 +1,32 @@
+- var navLinks = []
+- navLinks.push({ href: '/', text: 'webgl-operate' })
+- navLinks.push({ href: '/examples/', text: 'Examples', disabled: true })
```

Comment:
Just disabling it looks a bit weird - perhaps add `coming soon`?

Reply:
coming very soon ... integrated! 😄